src/coreComponents/mesh/PerforationData.hpp

```diff
@@ -177,6 +177,21 @@ class PerforationData : public ObjectManagerBase
    */
   arrayView1d< real64 > getWellTransmissibility() { return m_wellTransmissibility; }
 
+
+  /**
+   * @brief Provide an immutable accessor to a const perforation skin factor array.
+   * @return list of perforation well skin factors
+   */
+  arrayView1d< real64 const > getWellSkin() const { return m_wellSkin; }
+
+
+  /**
+   * @brief Get perforation well skin factors.
+   * @return list of perforation well skin factors
+   */
+  arrayView1d< real64 > getWellSkin() { return m_wellSkin; }
```

Comment:
is this getter needed?

Reply:
The dataflow that I got from doing this was:

1. XML input is passed into `Perforation`
2. `Perforation` passes to `InternalWellGenerator`
3. `InternalWellGenerator` passes to `LineBlock`
4. `LineBlock` passes to `WellElementSubRegion`
5. `WellElementSubRegion` passes to `PerforationData`

The `PerforationData::getWellSkin()` is needed as part of step 5. Hard setting the transmissibility at the perforation follows the same data flow.